packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// Identifiers for the supported Cupertino icons.
+///
+/// Use with with the [Icon] class to show specific icons.
+///
+/// Icons are identified by their name as listed below.
+///
+/// To use this class, make sure you add a dependency on `cupertino_icons` in your
+/// project's `pubspec.yaml` file. This ensures that the CupertinoIcons font is
+/// included in your application. This font is used to display the icons. For example:
+///
+/// ```yaml
+/// name: my_awesome_application
+///
+/// dependencies:
+///   cupertino_icons: ^0.1.0
+/// ```
+///
+/// See also:
+///
+///  * [Icon]
+class CupertinoIcons {
+  CupertinoIcons._();
+
+  // Manually maintained list
+
+  static const IconData left_chevron = const IconData(0xf3f0, fontFamily: 'packages/cupertino_icons/CupertinoIcons');
+
```

Comment:
I think IconData should have a 'package' parameter like TextStyle so that you don't need to write 'packages/...' in the font family name. I can do that and update this list.

Reply:
Thanks for the PR!